frontend/src/components/common/WeeklyEventItemPopUp.tsx

```diff
@@ -86,24 +89,28 @@ const WeeklyEventItemPopUp = ({
                   </HStack>
                   <HStack py="0.5rem">
                     <TimeIcon />
-                    <Text textStyle="desktopSmall">{convertTime(schedule!.startTime)} - {convertTime(schedule!.endTime)}</Text>
+                    <Text textStyle="desktopSmall">
+                      {convertTime(schedule!.startTime)} -{" "}
+                      {convertTime(schedule!.endTime)}
+                    </Text>
                   </HStack>
-                  {
-                    nextDropOff && (
-                      <Text textStyle="desktopSmall" py="0.5rem">
-                        Next Dropoff: {new Date(nextDropOff).toLocaleString(undefined, {
-                          year: "numeric",
-                          month: "long",
-                          day: "numeric",
-                        })}
-                      </Text>
-                    )
-                  }
+                  {nextDropOff && (
+                    <Text textStyle="desktopSmall" py="0.5rem">
+                      Next Dropoff:{" "}
+                      {new Date(nextDropOff).toLocaleString(undefined, {
+                        year: "numeric",
+                        month: "long",
+                        day: "numeric",
+                      })}
+                    </Text>
+                  )}
                 </VStack>
                 <Spacer />
                 <Badge
                   color={`${(colorMap as any)[schedule!.frequency]}.100`}
-                  backgroundColor={`${(colorMap as any)[schedule!.frequency]}.200`}
+                  backgroundColor={`${

```

Comment:
just curious but what is colorMap as any doing?

Reply:
I actually have no idea...I think this file change is in my PR bc `yarn fix` changed it but apparently it's this:
<img width="244" alt="Screen Shot 2021-12-01 at 9 42 37 PM" src="https://user-images.githubusercontent.com/37950626/144347931-9a43a9e9-a463-43f6-83ea-93a00effe599.png">

Reply:
it's to map the frequency to the appropriate badge colour!